docs/source/topics/data_types.rst

```diff
@@ -0,0 +1,177 @@
+.. _topics:data_types:
+
+**********
+Data types
+**********
+
+.. note:: Here comes datatypes (Marknik)
```

Comment:
Note sure if this typo is worth updating the code, since "Marknik" will be removing it soon anyway ;)
```suggestion
.. note:: Here comes datatypes (Marnik)
```